addons/gamification/models/res_users.py

```diff
@@ -12,13 +12,45 @@ class Users(models.Model):
     gold_badge = fields.Integer('Gold badges count', compute="_get_user_badge_level")
     silver_badge = fields.Integer('Silver badges count', compute="_get_user_badge_level")
     bronze_badge = fields.Integer('Bronze badges count', compute="_get_user_badge_level")
+    rank_id = fields.Many2one('gamification.karma.rank', 'Rank')
```

Comment:
I would put an index=False to say "we know it has no index and that we won't search on it as it is used for display". Same for next rank id.